### ISSUE TITLE
Quality of life: Creating new task in a todo list should immediately jump to edit task's name

### ISSUE THREAD

**Issue by magikmw** [Mad-Cookies-Studio/mad-productivity#32]
RIght now creating a new task requires:
* click of a button (could also be a keyboard shortcut)
* click on the title, preferably also select the "new_task" sample text
* type the name of a task

Could be:
* click a button
* start typing the name of a task
* enter to save, perhaps shift+enter to add a new one

**Comment by lentsius-bark** [comment]
agreed, this should improve usability indeed. unless you want to create 5 empty tasks first and then name them but that should be a more of a rarer case
